includes/Modules/AdSense.php

```diff
@@ -82,9 +82,18 @@ function( $data ) {
 			/**
 			 * Release filter forcing unlinked state.
 			 *
-			 * @see \Google\Site_Kit\Modules\Analytics\Settings::register
+			 * This is hooked into 'init' (default priority of 10), so that it
+			 * runs after the original filter is added.
+			 *
+			 * @see \Google\Site_Kit\Modules\Analytics::register()
+			 * @see \Google\Site_Kit\Modules\Analytics\Settings::register()
 			 */
-			remove_filter( 'googlesitekit_analytics_adsense_linked', '__return_false' );
+			add_action(
+				'init',
```

Comment:
We should probably hook this on `googlesitekit_init` which is still in the same action but should always run after everything else is loaded 👍 
```suggestion
				'googlesitekit_init',
```